test/integration/containerized/test_cleanup_images.py

```diff
@@ -0,0 +1,58 @@
+import shutil
+import json
+from base64 import b64decode
+import os
+
+import pytest
+
+from ansible_runner.cleanup import cleanup_images, prune_images
+from ansible_runner.defaults import default_container_image
+
+
+@pytest.mark.parametrize('runtime', ['podman', 'docker'])
+def test_cleanup_new_image(cli, runtime, tmp_path):
+    if shutil.which(runtime) is None:
+        pytest.skip(f'{runtime} is unavaialble')
+
+    # Create new image just for this test with a unique layer
+    special_string = "Verify this in test - 1QT4r18a7E"
+    dockerfile_path = str(tmp_path / 'Dockerfile')
+    with open(dockerfile_path, 'w') as f:
+        f.write('\n'.join([
+            'FROM {}'.format(default_container_image),
+            'RUN echo {} > /tmp/for_test.txt'.format(special_string)
+        ]))
+    image_name = 'quay.io/fortest/hasfile:latest'
+    build_cmd = [runtime, 'build', '--rm=true', '-t', image_name, '-f', dockerfile_path, str(tmp_path)]
```

Comment:
No need to prematurely convert this to a string.
```suggestion
dockerfile_path = tmp_path / 'Dockerfile'
dockerfile_path.write_text('\n'.join([
    'FROM {}'.format(default_container_image),
    'RUN echo {} > /tmp/for_test.txt'.format(special_string)
]))
image_name = 'quay.io/fortest/hasfile:latest'
build_cmd = [runtime, 'build', '--rm=true', '-t', image_name, '-f', str(dockerfile_path), str(tmp_path)]
```